### DISCUSSION TITLE
Add support for exporting medias along with messages

### DISCUSSION THREAD

**Discussion by M8inC** [d99kris/nchat#324] [Ideas]
First said, i like nchat very much and i'm using it on my root server to do regular backups of my "whatsapp business" Account. For me, it would be a great improvement, if the media files could also be saved when doing an --export?

**Comment by d99kris** [comment]
Hi @M8inC - thanks for the positive feedback 👍 

I personally use the `--export` option mainly to send its content to my own email once a year, to have all old conversations in plain text easily searchable. But I can see the benefit of having access to medias as well. What file format do you think would be useful in case we add this? HTML with inline attachments, or same txt format, just with the media files exported side-by-side, or something else?

**Reply by M8inC** [reply]
Thank you for considering the implementation!

For me it would be easiest if the media files were saved in their original format in the same folder as the messages. The media file name can be found in the export text file, so a reference can be easily established.

Short version: just save it ;)

**Reply by d99kris** [reply]
Ok thanks for the quick response 👍  I'll see when/if I can get a chance to look into this, will update here.
